internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -17,11 +17,52 @@
 
 package ssh
 
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/SUSE/skuba/pkg/skuba"
+	"github.com/pkg/errors"
+	"k8s.io/klog"
+)
+
 func init() {
+	stateMap["cri.configure"] = criConfigure
 	stateMap["cri.start"] = criStart
+	stateMap["cri.reset"] = crioReset
+}
+
+func criConfigure(t *Target, data interface{}) error {
+	criFiles, err := ioutil.ReadDir(skuba.CriDir())
+	if err != nil {
+		return errors.Wrap(err, "Could not read local cri directory: "+skuba.CriDir())
+	}
+	defer t.ssh("rm -rf /tmp/cri.d")
+
+	for _, f := range criFiles {
+		if err := t.target.UploadFile(filepath.Join(skuba.CriDir(), f.Name()), filepath.Join("/tmp/cri.d", f.Name())); err != nil {
+			return err
+		}
+	}
+
+	_, _, err = t.ssh("mv /etc/sysconfig/crio /etc/sysconfig/crio.backup")
```

Comment:
Maybe it would be better to use a `mv -f`, just in case the backup file is already there (and we don't really care about it)

Reply:
Done, thx :)